server/src/main/java/org/opensearch/index/ReplicationStats.java

```diff
@@ -93,5 +96,11 @@ static final class Fields {
         static final String MAX_BYTES_BEHIND = "max_bytes_behind";
         static final String TOTAL_BYTES_BEHIND = "total_bytes_behind";
         static final String MAX_REPLICATION_LAG = "max_replication_lag";
+        static final String MAX_DATA_LAG = "max_data_lag";
```

Comment:
nit - lets keep max_lag, total_lag?

Reply:
We can still preserve the original names. 

'max_bytes_behind_bytes'
'total_bytes_behind_bytes'
'max_replication_lag_millis'

and only show the existing fields if hr is off. If on the method will handle displaying them.

Reply:
What im suggesting would always show the existing fields. It would break compat when hr is set to true as the values would be displayed out of order.